cmd/kubeadm/app/cmd/join_test.go

```diff
@@ -17,13 +17,15 @@ limitations under the License.
 package cmd
 
 import (
+	"bytes"
 	"fmt"
 	"os"
 	"path/filepath"
+	"strings"
 	"testing"
 
 	"k8s.io/apimachinery/pkg/util/sets"
-
+	"k8s.io/klog/v2"
```

Comment:
should be under the apimachinery line above and have empty line below it.

Reply:
Done